protos/services/db_service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package services.db_service;
+
+import "db/query.proto";
+import "db/server.proto";
+import "db/pipeline.proto";
+
+service DBService {
+    rpc CreateStore (db.query.CreateStore) returns (db.server.Unit);
```

Comment:
Optional tracing id should be passed to every rpc call

Reply:
We've spoken that it would be a grpc header instead of getting inserted into the messages